azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1

```diff
@@ -1834,6 +1817,53 @@ function Deploy-AIO {
     }
 }
 
+function Deploy-ESA {
+    # increase the maximum number of files
+    Invoke-AksEdgeNodeCommand -NodeType "Linux" -Command 'echo 'fs.inotify.max_user_instances = 1024' | sudo tee -a /etc/sysctl.conf && sudo sysctl -p'
```

Comment:
This command needs to run on both cluster VMs

Reply:
Should we just use a busybox container running as root for configuration, and then run it as a daemon set?

Reply:
There was a typo in the line